starter_code/index.js

```diff
@@ -1 +1,17 @@
 const Elevator = require('./elevator.js');
+const Person = require('./person.js');
+
+let elevator = new Elevator();
+let person1 = new Person("Lam", 0, 4);
+let person2 = new Person("Ngoc", 0, 4);
+
+elevator.call(person1);
+elevator.call(person2);
+elevator.floorUp();
+elevator.update();
```

Comment:
You should be just calling `elevator.start()` here, which will start an interval and call `update` every second